packages/playwright-test/types/test.d.ts

```diff
@@ -3625,7 +3625,7 @@ interface LocatorAssertions {
 
     /**
      * When set to `"css"`, screenshot will have a single pixel per each css pixel on the page. For high-dpi devices, this will
-     * keep screenshots small. Using `"device"` option will produce a single pixel per each device pixel, so screenhots of
+     * keep screenshots small. Using `"device"` option will produce a single pixel per each device pixel, so screenshots of
```

Comment:
ℹ️ This is the only spelling error I corrected in non-docs code. It happened to contain the same typo that was in the docs when I did a find/replace, so I figured I'd fix it here too.